mssql-plugin/src/e2e-test/resources/pluginParameters.properties

```diff
@@ -8,6 +8,26 @@ username=MSSQL_USERNAME
 password=MSSQL_PASSWORD
 schema=dbo
 outputSchema=[{"key":"ID","value":"string"},{"key":"LASTNAME","value":"string"}]
+connectionArgumentsList=[{"key":"queryTimeout","value":"-1"}
+invalid.connectionArgumentsList=[{"key":"queryTimeout","value":"abc@!"}
+querytimeout=4
+logintimeout=3
+packetsize=1
+invalidDatabase=test123
+invalidImportQuery=select
+numberOfSplits=2
+invalidRef=invalidRef&^*&&*
+zeroValue=0
+blankvalue=
+invalidTableName=Table123@
+bqtarget.table=target-table
+project.id=cdf-athena
+datasetprojectId=cdf-athena
+invalid.boundQuery=SELECT MIN(id),MAX(id) FROM table
+splitby=id
+mssqlimportQuery=select * from SOURCETABLE_UUXHVMWPVL where $CONDITIONS;
```

Comment:
Don't hardcode the table name here, create a hook in TestSetup Hook and create the query there.  Then use PluginPropertyUtils.addPluginProp() to update this field ``` mssqlimportQuery ```

Reply:
resolved